src/analyser.cpp

```diff
@@ -1213,37 +1240,43 @@ void Analyser::AnalyserImpl::analyseEquationAst(const AnalyserEquationAstPtr &as
                         mModel->mPimpl->mVoi->mPimpl->populate(AnalyserVariable::Type::VARIABLE_OF_INTEGRATION,
                                                                0, nullptr, voi, nullptr);
                     }
-
-                    break;
                 }
-            }
-        } else if (!mModel->areEquivalentVariables(variable, mModel->mPimpl->mVoi->variable())) {
-            auto issue = Issue::IssueImpl::create();
+            } while (voi == nullptr);
+        } else {
+            // Note: see the llvm-cov section in src/README.rst for voiVariable.
```

Comment:
this seems unnecessary.

Reply:
same for all the other added instances of the same comment...don't think its a good idea to introduce cross-references like this as comments unless it really adds value.

Reply:
Addressed in 426cefe0.